WordPress/gutenberg
# Site Editor: Add View Link

Related #52632 

## What?

This PR continues the work on the great unification between post and site editors. This PR moves the ViewLink component from the edit-post package to the editor package and reuse the same panel in the site editor as well. This means that for post types that are viewable in the frontend, there will be a link to "view the page" in the frontend in the header of the editor in the site editor, similar to the post editor.

## Testing instructions

- Open a published page in the site editor, check that you can see a "view" link in the header next to the device previews.
